scripts/globals/status.lua

```diff
@@ -2654,6 +2654,7 @@ xi.mobMod =
     ATTRACT_FAMILY_NM         = 105, -- NMs within the same family will link onto this mob (used on Sabotenders for Cactrot Rapido)
     LEDGE_AGGRO               = 106, -- Used to increase vertical aggro range
     DISENGAGE_NO_PATH         = 107, -- Used to force a disengage when there is no vertical path to the target rather than despawn.
+    NO_CRYSTAL_SEAL_DROPS     = 108, -- Prevents a mob from dropping crystals or seals (to prevent pushing treasure out of pool for some mobs)
```

Comment:
Reality is the crystals should and seal should be last drops to not interfere with not getting the drops missed right?
Believe retail drops seals and crystals for sky gods right?

Reply:
> Reality is the crystals should and seal should be last drops to not interfere with not getting the drops missed right? Believe retail drops seals and crystals for sky gods right?

From reports of players it seems that the crystals and seal push out the non-rare/ex like crafting materials out of the pool, and this causes an issue, so this is like temp fix for that